cli/pkg/backend/aws/k8s_compute_backend.go

```diff
@@ -129,7 +129,30 @@ func (k8s *K8SComputeBackend) PrintLogs(ctx context.Context, stackName string, s
 			logrus.Error(err.Error())
 		}
 	}
-	return nil
+
+	if k8s.KubeConfig.AuthMethod != kube.AuthMethodEKS {
+		return nil
+	}
+
+	expression := fmt.Sprintf(`fields @timestamp, log
```

Comment:
When all stacks receive new annotations, will select by kubernetes lables, like `part-of` or `component`. For now, assuming pods are named `stackname-servicename`.